components/org.wso2.carbon.identity.api.server.authenticators/org.wso2.carbon.identity.api.server.authenticators.v1/src/main/resources/authenticators.yaml

```diff
@@ -0,0 +1,192 @@
+openapi: 3.0.0
+info:
+  version: "v1"
+  title: 'WSO2 Identity Server - Authenticators API Definition'
+  description: 'This document specifies a **RESTful API** for **WSO2 Identity Server Authenticators**'
+  contact:
+    name: WSO2
+    url: 'http://wso2.com/products/identity-server/'
+    email: architecture@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://localhost:9443/t/{tenant-domain}/api/server/v1'
+    variables:
+      tenant-domain:
+        default: carbon.super
+security:
+  - OAuth2: []
+  - BasicAuth: []
+
+paths:
+  /authenticators:
```

Comment:
Lets provide limit pagination support in the definition. We can implement them later. Currently we can throw "Not Implemented" Error.

Reply:
Added with cc03f33ddc1e99cbfa0c5359ecf6f978c663e707